SAPIO/donations.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- ticker: SAPIO
+  recipient:
+    name: SOCIAL HACKERS ACADEMY - ΣΟΣΙΑΛ ΧΑΚΕΡ ΑΣΤΙΚΗ ΜΗ ΚΕΡΔΟΣΚΟΠΙΚΗ ΕΤΑΙΡΙΑ (A.M.K.E.)
+    address: 3-5 Anaxagora Street, 10552 Athens Attica
+    tel: "+302103003124"
+    contact: damian@socialhackersacademy.org
+  eur: 100
```

Comment:
Hi guys, our CI requires an `ada` and a `usd` value and cannot deal with `eur` right now. Can you change this?